src/UI Component/Adminpanel/main.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+import ViewVerified from "./Activities/verifyloan/verifyloan";
+import CurrentLoan from "./Activities/currentloan";
+import AllLoanApplications from "./Activities/allloanapplication";
+import FirstPane from "../User/firstpane";
+import AdminHome from "./adminhome";
+import LoanApprovalPage from "./Activities/loanapproval"
+import SpecialLoanRequest from "./Activities/specialloans.js";
+import PostTransactionForm from "./Activities/postloanrepayment/posttransactionform";
+
+const Button = ({ name, activeButton, onClick }) => (
+  <button
+    className={`bg-blue-400 font-bold rounded-lg text-black px-4 py-2 mr-4 ${
+      activeButton === name && "bg-blue-700"
```

Comment:
This is a good approach.

I would say you use the [classnames](https://www.npmjs.com/package/classnames) library instead of using this method.